src/Illuminate/Foundation/Console/VendorPublishCommand.php

```diff
@@ -89,9 +89,8 @@ protected function determineWhatShouldBePublished()
             return;
         }
 
-        [$this->provider, $this->tags] = [
```

Comment:
Shouldn't this just become `list($this->provider, $this->tags) = [` to do the most minimal change possible to get the support level necessary?